minerl/data/download.py

```diff
@@ -17,30 +17,73 @@
 from minerl.data.version import DATA_VERSION, assert_version
 from minerl.herobraine import envs
 import tempfile
+from typing import Optional
 import coloredlogs
 
 logger = logging.getLogger(__name__)
 
 
-def download(directory=None, resolution='low', texture_pack=0,
-             update_environment_variables=True, disable_cache=False,
-             environment=None, competition=None):
-    """Downloads MineRLv0 to specified directory. If directory is None, attempts to 
-    download to $MINERL_DATA_ROOT. Raises ValueError if both are undefined.
-    
+def download(
```

Comment:
Oh good find -- that would be a confusing mismatch

Reply:
What about we put directory first?

It does seem useful to push the unused/internal arguments to the end.

Reply:
Here's the mismatching part:

![image](https://user-images.githubusercontent.com/1750835/125136481-c09fb700-e0bf-11eb-9d99-33437c666575.png)

Maybe the quickest way to fix this would just be removing this section from the "Installation" docs? There's already a ["Downloading and Sampling the Dataset" section](https://minerl.readthedocs.io/en/latest/tutorials/data_sampling.html) right underneath it in the user guide, and downloading the dataset isn't really part of installing MineRL.

Reply:
Directory first would probably be safe enough, yes.

Reply:
ok, I moved the directory arg to the front, and I removed the outdated docs I mentioned in the comment above.